cacheops/signals.py

```diff
@@ -0,0 +1,4 @@
+import django.dispatch
+
+post_queryset = django.dispatch.Signal(providing_args=["hit_cache"])
+post_cached_as = django.dispatch.Signal(providing_args=["function_name", "hit_cache"])
```

Comment:
Just use a single signal name it `cache_read` with `function` and `hit` arguments.